digestif.opam

```
@@ -52,13 +53,11 @@ depends: [
   "rresult"        {with-test}
   "ocamlfind"      {with-test}
 ]
-
 depopts: [
-  "ocaml-freestanding"
-  "mirage-xen-posix"
+  "ocaml-freestanding" {>= "0.6.0"}
```

```suggestion
  "ocaml-freestanding"
```

this should not be needed, below there's a conflict with ocaml-freestanding < 0.4.3 --- which should be sufficient for your needs. what is the error you're trying to solve with the >= 0.6.0 constraint?

---

please disregard my comment, indeed `__ocaml_freestanding__` is only defined since 0.6.0 -- the suggestion above is still valid though :)